src/dftbp/dftb/sparse2dense.F90

```diff
@@ -3227,4 +3227,41 @@ subroutine getSparseDescriptor(iNeighbour, nNeighbourSK, img2CentCell, orb, iPai
 
   end subroutine getSparseDescriptor
 
+
+  !> Calculates number of elements in sparse arrays like the real space overlap.
```

Comment:
Worth commenting this is the single triangle neighbour case?

Reply:
Shouldn't this routine work for both cases? Or am I missing something?

Reply:
I renamed `nNeighbourSK` $\rightarrow$ `nNeighbour` (also in `getSparseDescriptor` above) to indicate that both routines work for the symmetric and asymmetric neighbour-list.